scripts/repo_health.sh

```diff
@@ -53,5 +53,7 @@ shellcheck -s bash -ax \
   scripts/*.sh \
   integration/**/*.sh
 
+shellcheck -s bash -ax -S error .studiorc .studio/*
```

Comment:
❓ Worth a comment that this is not in the list above?

Reply:
Good call. I'm about to change this in a follow-up PR to actually just be the list of checks that we fail since that will make it a sort of todo list:

```
> shellcheck -s bash -ax --format=gcc .studiorc .studio/* | awk '{print $NF}' | sort | uniq -c
   1 [SC1010]
   6 [SC2004]
   3 [SC2012]
   5 [SC2034]
   4 [SC2046]
   1 [SC2048]
 110 [SC2086]
   1 [SC2089]
   1 [SC2090]
   3 [SC2119]
   3 [SC2120]
   2 [SC2124]
   2 [SC2128]
   5 [SC2154]
   1 [SC2162]
  37 [SC2164]
   9 [SC2181]
   4 [SC2207]
   1 [SC2231]
   5 [SC2236]
```

So, I promise I'll comment this in the next PR, right now I'm impatient and don't want to wait for tests again.